client/utils/account-fees.tsx

```diff
@@ -359,7 +359,10 @@ export const getTransactionsPaymentMethodName = (
 		case 'affirm':
 			return __( 'Affirm transactions', 'woocommerce-payments' );
 		case 'afterpay_clearpay':
-			return __( 'Afterpay transactions', 'woocommerce-payments' );
+			return __(
+				'Afterpay / Clearpay transactions',
```

Comment:
`Afterpay` I guess?

Reply:
Fixed here ( Sorry. I changed it in my other PR and forgot to do the same here :D ) 
a0999f0